package.json

```diff
@@ -74,7 +74,7 @@
     "redux-thunk": "^2.2.0",
     "vega": "github:vega/vega#v3.2.1",
     "vega-datasets": "^1.18.0",
-    "vega-lite": "github:vega/vega-lite#v2.3.0",
+    "vega-lite": "^v2.3.1",
```

Comment:
This doesn't work because of the examples.

Reply:
Oh! What the possible solution then?

Reply:
Use GitHub but use the latest version.

Reply:
I've updated it to `github:vega/vega-lite#v2.3.1` & it's working fine.